### PR TITLE
feat: miscellaneous enhancements to the bot

### LINKED ISSUE #9: Allow issue numbers in commit description (but not commit message)
My understanding from talking with @erictleung is that currently Commitizen allows issue numbers in the commit descriptions, but not in the commit message itself.

CamperBot will give contributors a warning if they create an issue with an issue number in the description. We should follow Commitizen's convention and allow issue numbers in the commit descriptions, though not in the commit message itself.

### PR DESCRIPTION
**Pull request by raisedadead**
- Have updated the configs and the bot will not cry about the closes keyword now.
- Added the commitizen and standard commits goodies.
- Updated the web view of the bot.

Closes #9

### CODE REVIEW

**Comment by raisedadead** [comment]
@BerkeleyTrue I have done some updates to the bot, and will be making more in future when I have more time at hand. 

For now, this can go in as it is. I have updated the PR OP to explain whats added for now.

Can you take a look and deploy if this is ok.

File: package.json [hunk 1]
```diff
@@ -6,12 +6,18 @@
   "scripts": {
     "start": "node run-server.js",
     "test": "./node_modules/.bin/babel-node node_modules/mocha/bin/_mocha --recursive -R spec",
-    "coverage": "./node_modules/.bin/babel-node ./node_modules/istanbul/lib/cli cover -- node_modules/mocha/bin/_mocha --recursive --reporter spec"
+    "coverage": "./node_modules/.bin/babel-node ./node_modules/istanbul/lib/cli cover -- node_modules/mocha/bin/_mocha --recursive --reporter spec",
```
Comment (BerkeleyTrue) [inline package.json]: You don't need to fully qualify the path of a local module. npm will automatically create a global symbol local bin files while running scripts

**Review by BerkeleyTrue — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
@BerkeleyTrue I have actually gone ahead and removed the old test and coverage configs since they aren't working right now, we already have an issue to track those, and I plan to work on them.

**Comment by BerkeleyTrue** [comment]
👍 

Is this ready for review?

**Review by BerkeleyTrue — APPROVED** [review]

**Comment by raisedadead** [comment]
Ping @BerkeleyTrue I guess we could merge this and deploy.

**Pull request merged**